controllers/controller.js

```diff
@@ -1,10 +1,47 @@
-const { getAllTopics: getTopics } = require("../models/models");
+const {
+  getAllTopics,
+  getAllEndpoints,
+  getArticlesByID,
+} = require("../models/models");
+const endpoints = require("../endpoints.json");
 
 exports.getTopics = (req, res, next) => {
-  getTopics().then((rows) => {
-    console.log(rows);
-    res.status(200).send({ topics: rows });
-  }).catch((err) => {
-    res.status(err.status).send({ msg: err.message });
-  })
-};
\ No newline at end of file
+  getAllTopics()
+    .then((rows) => {
+      res.status(200).send({ topics: rows });
+    })
+    .catch((err) => {
+      res.status(err.status).send({ msg: err.message });
+    });
```

Comment:
These errors should ideally be extracted out from here and handled in some error handling middleware functions